spring-cloud-stream-binders/spring-cloud-stream-binder-spi/src/main/java/org/springframework/cloud/stream/binder/MessageChannelBinderSupport.java

```diff
@@ -824,7 +828,7 @@ protected boolean isNamedChannel(String name) {
 	}
 
 	/**
-	 * Attempt to create a direct binding (avoiding the bus) if the consumer is local. Named channel producers are not
+	 * Attempt to create a direct binding (avoiding the binder) if the consumer is local. Named channel producers are not
```

Comment:
I don't think "avoiding the binder" sounds right, since we're talking about direct *binding*... maybe this should change to avoiding a "broker"